tests/utils/test_backbone_utils.py

```diff
@@ -16,11 +16,22 @@
 
 import pytest
 
+from transformers import MaskFormerConfig
+from transformers.testing_utils import require_torch, slow
 from transformers.utils.backbone_utils import (
     BackboneMixin,
     get_aligned_output_features_output_indices,
+    load_backbone,
     verify_out_features_out_indices,
 )
+from transformers.utils.import_utils import is_torch_available
+
+
+if is_torch_available():
+    import torch
+
+    # from transformers import PretrainedModel
+    from transformers import BertPreTrainedModel as PretrainedModel
```

Comment:
could you explain why we need this?

Reply:
I've changed this to just use `BertPreTrainedModel` - this was just something I didn't tidy up, sorry. 

`BertPreTrainedModel` is imported for the weight init for the dummy model used in tests